Python/Music_Genre_Classification/README.md

```diff
@@ -0,0 +1,153 @@
+# Music_Genre_Classification
+
+Music genre classification is one of the most basic yet fascinating DL/ML project. People use different approaches
+ranging from basi classification techniques like Survey and questionnaire to highly advanced combinations of Machine
+Learning Classifiers. The script that follows is a very basic implementation of one such model, revolving around Deep 
+Learning, __Sequential Model__.
+
+This script will not only enable you to setup a basic classification system, but will also help in feature extraction,
+dataset division, training  and testing. Please be advised, that the aim is not to setup a really advanced
+classification system, but rather a simple setup, using a renowned dataset, to predict the genre of a musical piece
+using minimal features.
+
+## Dependency
+
+The dependencies are summed uup in [`requirements.txt`](requirements.txt).
+
+A separate list is also available below:
+
+- librosa
+- csv
+- Keras
+- TensorFlow
+- numba
+- numpy
+- pandas
+- scikit-learn
+- scipy
+
+Considering there are some big libraries here, it is recommended to setup a virtual environment first. This
+[link](https://realpython.com/lessons/creating-virtual-environment/) might help.
+
+## Dataset
+
+We have used GTZAN dataset for our task. It is publicly available, here is the 
```

Comment:
Since you have already referenced the GTZAN Dataset, do we actually need to store it in the repo as well?

Reply:
I don't get that, I have just referenced the Repo here, and have stored a file containing 6 features, the whole DB is 1.2 gb.

Can you clarify this?

Reply:
The size of the `csv` is also half MB, if you want I can get rid of the file and paste a drive link there

Reply:
Yeah. The only intent was to get rid of the files that can be retrieved from online sources.